Task05 1.

gemansipator opened the pull request:
1. Считать 6 строк и заполнить ими массив strings.
 2. Удалить повторяющиеся строки из массива strings,
 заменив их на null (null должны быть не строками "null")

gemansipator committed:
Task05 1. Считать 6 строк и заполнить ими массив strings.
        2. Удалить повторяющиеся строки из массива strings,
        заменив их на null (null должны быть не строками "null")

gemansipator committed:
Task05 1. Считать 6 строк и заполнить ими массив strings.
        2. Удалить повторяющиеся строки из массива strings,
        заменив их на null (null должны быть не строками "null")

RinatBeybutov commented on src/main/java/site/javadev/lesson_06/homework/Task05.java:
```diff
@@ -0,0 +1,36 @@
+            if (str[i] == null) {
+                continue; // Пропускаем, если элемент уже null
+            }
+            for (int j = i + 1; j < str.length; j++) {
```
этот алгоритм работает некорректно, если есть 3 одинаковые строки, то занулятся только 2 из них, а должны все. Чуть-чуть нужно доработать алгоритм

gemansipator replied:
ок

gemansipator committed:
Task05 1. Считать 6 строк и заполнить ими массив strings.
        2. Удалить повторяющиеся строки из массива strings,
        заменив их на null (null должны быть не строками "null")

RinatBeybutov commented on src/main/java/site/javadev/lesson_06/homework/Task05.java:
```diff
@@ -0,0 +1,36 @@
+            if (str[i] == null) {
+                continue; // Пропускаем, если элемент уже null
+            }
+            for (int j = i + 1; j < str.length; j++) {
```
нужно тыкать, что этот тред разрешен. Тут где-то галка должна быть

RinatBeybutov commented:
ревью принято
